### PR TITLE
Add default category

### LINKED ISSUE #2552: Add default category
When the system is initially installed all categories need to include a default category "Uncategorized". Add to the seeder

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
Resolves #2552 

- Add seeder for:
- processes
- screens
- scripts

### CODE REVIEW

File: database/seeds/CategorySystemSeeder.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+class CategorySystemSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $name = __('Uncategorized');
```
Comment (velkymx) [inline database/seeds/CategorySystemSeeder.php]: This shouldn't translate...

File: database/seeds/DatabaseSeeder.php [hunk 2]
```diff
@@ -21,7 +21,8 @@ public function run()
             ProcessSystemCategorySeeder::class,
             GroupSeeder::class,
             ScreenTypeSeeder::class,
-            ScreenSystemSeeder::class
+            ScreenSystemSeeder::class,
```
Comment (velkymx) [inline database/seeds/DatabaseSeeder.php]: Do we need all of these seeders with this change?

Comment (marcoAntonioNina) [reply database/seeds/DatabaseSeeder.php]: The default screen is used in Feature interstitial

**Comment by marcoAntonioNina** [comment]
@velkymx comments fixed.

**Pull request merged**